src/state/queries/post-thread.ts

```diff
@@ -360,7 +359,7 @@ function responseToThreadNodes(
         depth,
         isHighlightedPost: depth === 0,
         hasMore:
-          direction === 'down' && !node.replies?.length && !!node.replyCount,
+          direction === 'down' && !node.replies?.length && !!post.replyCount,
```

Comment:
I think this was actually a typo, which was masked by the `[key: string]: undefined` loose types we use to have here. `node` does not have a `replyCount` on it at all, but `node.post` does.